src/main/java/access/messaging/AccessThreadManager.java

```diff
@@ -65,134 +54,31 @@
-			WorkerCallback callback = jobId -> runningJobs.remove(jobId);
-
-			// Create the General Group Consumer
-			generalConsumer = KafkaClientFactory.getConsumer(kafkaAddress, kafkaGroup);
-			List<String> topics = Arrays.asList(String.format(LOGGER_FORMAT, ACCESS_TOPIC_NAME, space));
-			// Log topics we're listening to
-			pzLogger.log(String.format("Begin listening to Kafka topics : %s", String.join(", ", topics)), Severity.INFORMATIONAL);
-			// Listen
-			generalConsumer.subscribe(topics);
-
-			// Poll
-			while (!closed.get()) {
-				ConsumerRecords<String, String> consumerRecords = generalConsumer.poll(1000);
-				// Handle new Messages on this topic.
-				for (ConsumerRecord<String, String> consumerRecord : consumerRecords) {
-					pzLogger.log(String.format("Processing Job ID %s on Access Worker Thread.", consumerRecord.key()),
-							Severity.INFORMATIONAL);
-					// Create a new worker to process this message and add it to
-					// the thread pool.
-					Future<?> workerFuture = accessWorker.run(consumerRecord, producer, callback);
-
-					// Keep track of all Running Jobs
-					runningJobs.put(consumerRecord.key(), workerFuture);
-				}
-			}
-		} catch (WakeupException exception) {
-			String error = String.format("Polling Thread forcefully closed: %s", exception.getMessage());
-			LOGGER.error(error, exception, new AuditElement("access", "kafkaListenerShutDown", ""));
-			pzLogger.log(error, Severity.ERROR);
-		}
-		finally {
-			if( generalConsumer != null ) {
-				generalConsumer.close();
-			}
-		}
-	}
 
 	/**
-	 * Begins listening for Abort Jobs. If a Job is owned by this component, then it will be terminated.
+	 * Processes an Abort Job message coming in through the queue. If this component contains that job, it will cancel
+	 * it.
+	 * 
+	 * @param abortJobRequest
+	 *            The request containing the information about the Job cancellation request
 	 */
-	public void pollAbortJobs() {
-		
-		Consumer<String, String> uniqueConsumer = null;
-		
+	@RabbitListener(queues = "Abort-Job-${SPACE}")
+	public void processAbortJob(String abortJobRequest) {
+		String jobId = null;
 		try {
-			// Create the Unique Consumer
-			uniqueConsumer = KafkaClientFactory.getConsumer(kafkaAddress,
-					String.format(LOGGER_FORMAT, kafkaGroup, UUID.randomUUID().toString()));
-			uniqueConsumer.subscribe(Arrays.asList(String.format(LOGGER_FORMAT, JobMessageFactory.ABORT_JOB_TOPIC_NAME, space)));
-
-			// Poll
-			while (!closed.get()) {
-				ConsumerRecords<String, String> consumerRecords = uniqueConsumer.poll(1000);
-				// Handle new Messages on this topic.
-				for (ConsumerRecord<String, String> consumerRecord : consumerRecords) {
-					handleNewMessages(consumerRecord);
-				}
-			}
-		} catch (WakeupException exception) {
-			String error = String.format("Polling Thread forcefully closed: %s", exception.getMessage());
+			PiazzaJobRequest request = objectMapper.readValue(abortJobRequest, PiazzaJobRequest.class);
```

Comment:
Consider using the keyword `final` for all variables that do not get reassigned, particularly method parameters.